From 6054eb157a25b4b46427baa6eb53b5e18135019a Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz93@gmail.com>
Date: Sun, 19 May 2024 21:15:34 -0400
Subject: [PATCH 1/2] fix quoting around potentially empty shell var

required to handle POSIX `test` command

 * QA Notice: Broken syntax in configure script
 *
 * 	./configure: line 8046: test: =: unary operator expected
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index d4f3a44..2a9b0ff 100644
--- a/configure.in
+++ b/configure.in
@@ -179,7 +179,7 @@ AC_CHECK_MEMBERS([struct stat.st_blksize])
 AC_CHECK_MEMBERS([struct stat.st_rdev])
 
 AC_FUNC_FSEEKO
-if test $ac_cv_func_fseeko = no; then
+if test "$ac_cv_func_fseeko" = no; then
     AC_LIBOBJ(fseeko)
     AC_LIBOBJ(ftello)
 fi
-- 
2.44.1

